src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -31,8 +38,7 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
         private readonly SynthesizedBackingFieldSymbol _backingField;
         private readonly TypeSymbol _explicitInterfaceType;
         private readonly ImmutableArray<PropertySymbol> _explicitInterfaceImplementations;
-        private readonly bool _isExpressionBodied;
-        private readonly bool _isAutoProperty;
+        private readonly SourcePropertySymbolFlags _propertyFlags;
         private readonly RefKind _refKind;
```

Comment:
i think having these two bytes next to each other is technically wasteful in .net.  We could consider merging into a single-shared byte.  But i don't know if that's valuable enough and/or considerd too-cute.  The above code should at least be slightly better in terms of making the two bools into a single field.

Reply:
I'm open to it. If you do so consider including other fields that seem appropriate such as _modifiers. There's precedent for this here (you may already be familiar?): http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/Source/SourceMemberMethodSymbol.cs,20

Reply:
It's up to you guys :)  LMK if you want this and i'll do it.

Reply:
Personally, I don't care enough about a single byte. That feels like an overoptimization, let's have the code be clear here.